### PR TITLE
Update service meta data

### PR DESCRIPTION
**Pull request by msecret**
In order to have more consistency in service names, Navigator is updating the service names as they appear in marketplace and other places.

Part of this story: 18F/cg-dashboard#857

### CODE REVIEW

File: catalogData/elasticsearch17/1x/plan.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "id": "809fc3ae-1f0c-4e57-acc9-3d1d567d73aa",
   "name": "1x",
-  "description": "1G Memory / 1x CPU",
+  "description": "Elasticsearch instance with 1GB of RAM and 1 slices of CPU",
```
Comment (thisisdano) [inline catalogData/elasticsearch17/1x/plan.json]: `slices` s/b `slice` (singular)

File: catalogData/elasticsearch23/1x/plan.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "id": "dafd5d6e-0656-4a7d-b23b-3c4fb9b13d01",
   "name": "1x",
-  "description": "1G Memory / 1x CPU",
+  "description": "Elasticsearch instance with 1GB of RAM and 1 slices of CPU",
```
Comment (thisisdano) [inline catalogData/elasticsearch23/1x/plan.json]: `slices` s/b `slice`

**Review by thisisdano — COMMENTED** [review]
Just two quick typos, otherwise OK.

File: catalogData/redis28/standard/plan.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "id": "30da8e96-16a9-11e6-b43f-00155d3d8807",
   "name": "standard",
-  "description": "standard",
+  "description": "Redis instance with unlimited storage.",
```
Comment (jmcarp) [inline catalogData/redis28/standard/plan.json]: Storage isn't unlimited 😄

**Review by jmcarp — DISMISSED** [review]

**Review by sharms — APPROVED** [review]

**Comment by mogul** [comment]
Sorry I'm late to the party... What's the reasoning for adding "for GovCloud"?

**Comment by thisisdano** [comment]
I don't remember exactly — it might have come from some now-deprecated copy in the Services documentation. Likely out of date, if ever correct.

**Pull request merged**